flintrock/templates/spark/conf/spark-defaults.conf

```diff
@@ -0,0 +1 @@
+spark.jars.packages    org.apache.hadoop:hadoop-aws:{hadoop_version}
```

Comment:
I usually add `com.amazonaws:aws-java-sdk:1.7.4` along `org.apache.hadoop:hadoop-aws:2.7.3`, e.g. `spark-submit --packages "com.amazonaws:aws-java-sdk:1.7.4,org.apache.hadoop:hadoop-aws:2.7.3"` or `spark-shell --packages "com.amazonaws:aws-java-sdk:1.7.4,org.apache.hadoop:hadoop-aws:2.7.3"`.

I tried launching spark-shell on my laptop, with `org.apache.hadoop:hadoop-aws:2.7.3` but without `com.amazonaws:aws-java-sdk:1.7.4`, it raised the following exception:
```
java.lang.ClassNotFoundException: com.amazonaws.auth.AWSCredentialsProvider
	at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 101 more
```